src/modules/python.rs

```diff
@@ -76,12 +76,19 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
 }
 
 fn get_pyenv_version(context: &Context) -> Option<String> {
-    let version_name = context
-        .exec_cmd("pyenv", &["version-name"])?
-        .stdout
-        .trim()
-        .to_string();
-    Some(version_name)
+    let mut version_name = context.get_env("PYENV_VERSION");
+
+    if version_name.is_none() {
+        version_name = Some(
+            context
+                .exec_cmd("pyenv", &["version-name"])?
+                .stdout
+                .trim()
+                .to_string(),
+        )
+    }
+
+    version_name
```

Comment:
Not a blocker but I think this can be simplified to
```suggestion
    match context.get_env("PYENV_VERSION") {                                                                                
        Some(version) => Some(version),                                                                                     
        _ => Some(                                                                                                          
            context                                                                                                         
                .exec_cmd("pyenv", &["version-name"])?                                                                      
                .stdout                                                                                                     
                .trim()                                                                                                     
                .to_string(),                                                                                                                                                                                     
        ),                                                                                                                  
    }
```